MANAGEMENT_NETWORK.md

```diff
@@ -0,0 +1,318 @@
+# The Octavia Amphora Management Network
+
+The Octavia Amphora controller pods require network connectivity across the
+OpenStack cloud in order to monitor and manage amphora Load Balancer VMs . This
+document describes the elements that implement the management network.
+
+The _Octavia management network_ is actually two OpenStack networks: a tenant
```

Comment:
Provider vs.  tenant networks is a bit of a fuzzy conversation, more that can really go into - there is a breakdown here: https://superuser.openinfra.dev/articles/spot-the-difference-tenant-provider-and-external-neutron-networks/  by our still-missed former colleague Assaf Muller.  While it is indicated later, I should add the extra qualifier "external" to this reference to the provider network here. 

I did consider the inclusion of the data plane but made the diagram too wide but I'll see if I can make it make sense by making it bigger vertically and put the dataplane below the OpenShift workers. I had a go at a diagram intro at one point as well but scrapped it as it was difficult to make clear to a general reader. I'll have another crack at it.

Reply:
Thanks. On the internet there are different definitions of the terms, which led to my confusion, and isn't really a fault of this document. Daniel has sent me a link to another blog post after reading my comment: https://assafmuller.com/2018/07/23/tenant-provider-and-external-neutron-networks/ which I found helpful as well. A provider network does not necessarily need to be a physical network.

Maybe the data plane can be a single block in the diagram if it gets too complex otherwise.